Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -8,6 +8,12 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
+    public enum GraspPointPlacement
+    {
+        BetweenFingerAndThumb,
+        Fingertip,
```

Comment:
Should we update these to specific what `Finger` is referring to? Something like `BetweenIndexFingerAndThumb` and `IndexFingertip`

Reply:
And maybe something to specific that `BetweenIndexFingerAndThumb` is referring to the tips too, but that might get a little too verbose...

Reply:
Done! I think those are better names